app/scripts/proactive/config.js

```diff
@@ -6,6 +6,7 @@ define(function () {
     
     return {
         restApiUrl: '/rest/studio',
+        execution_scheduler_restApiUrl: '/execution-scheduler/scheduled-workflows',
```

Comment:
Can we make this configurable with an IP/Hostname. The service could run on another machine.